mkdocs.yml

```diff
@@ -0,0 +1,61 @@
+site_name: ColorHighlighter Documentation
+site_url: https://Monnoroch.github.io/ColorHighlighter
+repo_url: https://github.com/Monnoroch/ColorHighlighter
+edit_uri: tree/master/docs/src
+site_description: A color highlighting plugin for Sublime Text 2 and 3.
+
+pages:
+  - ColorHighlighter: index.md
+
+theme: material
+docs_dir: docs/src
+theme_dir: docs/theme
+
+markdown_extensions:
+  - markdown.extensions.toc:
+      slugify: !!python/name:pymdownx.slugs.gfm
+  - markdown.extensions.admonition:
+  - markdown.extensions.smarty:
+      smart_quotes: false
+  - pymdownx.betterem:
+  - markdown.extensions.attr_list:
+  - markdown.extensions.def_list:
+  - markdown.extensions.tables:
+  - markdown.extensions.abbr:
+  - pymdownx.extrarawhtml:
+  - pymdownx.superfences:
+  - pymdownx.highlight:
+      css_class: codehilite
+  - pymdownx.inlinehilite:
+  - pymdownx.magiclink:
+      repo_url_shortener: true
+      base_repo_url: https://github.com/facelessuser/ColorHighlighter
```

Comment:
Is it correct?

Reply:
Nope.  I copied this from one my repos and just missed it.  It should be Monnoroch.